### DISCUSSION TITLE
Unable to login into UI with multiple replica count

### DISCUSSION THREAD

**Discussion by micrOsman** [provectus/kafka-ui#3860] [Q&A] [answered]
Hello all,

I have trouble regarding multiple Kafka-UI replica counts. I'm not able to login into Kafka UI when more than 1 replica count is set. The authentication method is Basic Authentication - username+password. When I fill in the correct credentials login page just reload and no action happens. No events in pod logs are visible. Do you have any idea?

Thank you.

App version: v0.7.0

Kafka-UI helm config:

```USER-SUPPLIED VALUES:
envs:
  config:
    AUTH_TYPE: LOGIN_FORM
    KAFKA_CLUSTERS_0_BOOTSTRAPSERVERS: xxx:9092,yyy:9092,zzz:9092
    KAFKA_CLUSTERS_0_NAME: kafka
    KAFKA_CLUSTERS_0_READONLY: "true"
    KAFKA_CLUSTERS_0_ZOOKEEPER: xxx:2181,yyy:2181,zzz:2181
    KAFKA_CLUSTERS_0_ZOOKEEPER_TLS: xxx:2182,yyy:2182,zzz:2182
    SERVER_SERVLET_CONTEXT_PATH: ""
    SPRING_SECURITY_USER_NAME: admin
    SPRING_SECURITY_USER_PASSWORD: passwd
  secret: {}
image: {}
ingress:
  enabled: true
  host: hostname-value
  tls:
    enabled: true
    secretName: ""
replicaCount: 2
resources:
  requests:
    cpu: 200m
    memory: 256Mi
```

**Comment by vanabbott** [comment, marked as answer]
I was running into this same issue with my setup and was able to fix by enabling sticky sessions with the ingress deployed with application [example](https://zhimin-wen.medium.com/sticky-sessions-in-kubernetes-56eb0e8f257d)

Essentially just need to add the following annotations to your ingress: 
    nginx.ingress.kubernetes.io/affinity: "cookie"
    nginx.ingress.kubernetes.io/session-cookie-name: "route"
    nginx.ingress.kubernetes.io/session-cookie-expires: "172800"
    nginx.ingress.kubernetes.io/session-cookie-max-age: "172800"
